torchaudio/datasets/dr_vctk.py

```diff
@@ -0,0 +1,109 @@
+from pathlib import Path
+from typing import Dict, Tuple, Union
+
+from torch import Tensor
+from torch.utils.data import Dataset
+
+import torchaudio
+from torchaudio.datasets.utils import (
+    download_url,
+    extract_archive,
+)
+
+
+URL = "https://datashare.ed.ac.uk/bitstream/handle/10283/3038/DR-VCTK.zip"
+FOLDER_IN_ARCHIVE = Path("DR-VCTK") / "DR-VCTK"
+_CHECKSUMS = {
+    "https://datashare.ed.ac.uk/bitstream/handle/10283/3038/DR-VCTK.zip": "29e93debeb0e779986542229a81ff29b",
+}
+
+
+class DR_VCTK(Dataset):
+    """Create a dataset for Device Recorded VCTK (Small subset version).
+
+    Args:
+        root (str or Path): Root directory where the dataset's top level directory is found.
+        download (bool):
+            Whether to download the dataset if it is not found at root path. (default: ``False``).
+        url (str): The URL to download the dataset from.
+            (default: ``"https://datashare.ed.ac.uk/bitstream/handle/10283/3038/DR-VCTK.zip"``)
+        train (bool): If True, creates dataset from training set, otherwise creates from test set.
```

Comment:
In case there is a version update with an addition of new subset in the future, can we change the signature to be more extendable one?
I think `"subset (str)"` with valid value of `"train"` and `"test"` is straight forward choice.